client/src/app/modules/sidebar/components/sidebar.component.html

```diff
@@ -4,6 +4,7 @@
 
 <ng-container *ngIf="isError" data-test-error>
   <div class="error">The project list could not be loaded currently</div>
+  <!-- TODO add a reload button -->
```

Comment:
Usually this kind of error should never happen. If it happens frequently, something is broken and that something might also cause other thing to fail frequently. Since the whole application might be unstable, I would not integrate the error handling like this. I would rather show a global error, e.g. via popup or switching to an error page.

Reply:
In the previous implementation I thought it can happen that one endpoint works but the other not - guess this is not realistic, either the endpoints are all responding or using the app is pretty much senseless. So now the app component tries to get all projects and if this doesn't work a general error page will be shown.